### ISSUE TITLE
UI for default project quotas

### ISSUE THREAD

**Issue by lsviben** [kubermatic/dashboard#5479]
Add an UI for default project quotas.

It should be somewhere in the admin panel, along with the other defaults settings. Maybe Limits tab?

the default project quota is a part of the KubermaticSetting, so it can be managed using the KubermaticSettings endpoint

**Comment by Waseem826** [comment]
Is this same as https://github.com/kubermatic/dashboard/issues/4860? 
Also, is API part ready for this?

**Comment by lsviben** [comment]
> Is this same as #4860? Also, is API part ready for this?

Right, i missed it, but that one is the design, so disregard my input where we are putting this, its better where @cschieder placed it. This ticket would be the implementation in the UI.

The API is ready

**Comment by cschieder** [comment]
Should we close one of the issues?
